apps/address-book/app/components/App/App.js

```diff
@@ -40,12 +40,14 @@ const App = () => {
 
   return (
     <Main assetsUrl={ASSETS_URL}>
-      <Header
-        primary="Address Book"
-        secondary={
-          <Button mode="strong" icon={<IconPlus />} onClick={newEntity} label="New Entity" />
-        }
-      />
+      { entries.length > 0 && (
```

Comment:
Yes. That makes a lot of sense.

Reply:
OK. I consolidated the `if no entries` logic into `App.js`. However, I decided not to do it in two different `return` blocks as you propose, because when there are no entries you still need the `SidePanel`, so the code ended up being redundant. Instead, I made three "inline" checks.

In other words, I'm sacrificing clarity for succinctness. Let me know what you think.